phoenicis-javafx/src/main/java/org/phoenicis/javafx/views/mainwindow/apps/ViewApps.java

```diff
@@ -79,13 +81,28 @@ public void showAvailableApps() {
      */
     public void populate(List<CategoryDTO> categories) {
         Platform.runLater(() -> {
+            Set<String> themedCategories = Sets.newHashSet(
+                    "custom",
```

Comment:
I don't understand what is your blocker here:
What not asking ThemeManager if a resource with the name of the icon exists. If yes, then we can use it. 
It not, use the repository one